File: Leetcode/266/266.cpp
class Solution {
public:
    bool canPermutePalindrome(string s) {
        constexpr int MAXN = 26;
        int stats[MAXN]{};
        for (const auto &i : s) {
            ++stats[i - 'a'];
        }
        int counter = 0;
        for (int i = 0; i < MAXN; ++i) {
            if (stats[i] % 2 != 0) {
                ++counter;
                if (counter > 1) {
                    return false;
                }
            }
        }
        return true;
    }
};

File: Leetcode/266/266.go
func canPermutePalindrome(s string) bool {
	const MAXN = 26
	stats := [MAXN]int{}
	for i := 0; i < len(s); i++ {
		stats[s[i] - 'a']++
	}
	counter := 0
	for i := 0; i < MAXN; i++ {
		if (stats[i] & 1) != 0 {
			counter++
			if counter > 1 {
				return false
			}
		}
	}
	return true
}